gateway/api/models.py

```diff
@@ -13,6 +17,39 @@ def get_upload_path(instance, filename):
     return f"{instance.author.username}/{instance.id}/{filename}"
 
 
+class JobConfig(models.Model):
+    """Job Configuration model."""
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    created = models.DateTimeField(auto_now_add=True)
+
+    auto_scaling = models.BooleanField(default=False, null=True)
+    workers = models.IntegerField(
+        null=True,
+        validators=[
+            MinValueValidator(0),
+            MaxValueValidator(settings.RAY_CLUSTER_WORKER_REPLICAS_MAX),
```

Comment:
configuration values in models will create conflict. If I change my settings now it will generate new migration diff and django will be complaining about migrations.